packages/block-library/src/gallery/block.json

```diff
@@ -41,12 +44,20 @@
 		"isListItem": {
 			"type": "boolean",
 			"default": true
+		},
+		"applyToAllImages": {
+			"type": "boolean",
+			"default": false
 		}
 	},
 	"providesContext": {
 		"allowAlign": "allowAlign",
 		"allowResize": "allowResize",
-		"isListItem": "isListItem"
+		"isListItem": "isListItem",
+		"applyToAllImages": "applyToAllImages",
+		"galleryLinkDestination": "linkTo",
```

Comment:
parentLinkDestination, etc. may be better than galleryLink... that way something that isn't strictly a gallery could also make use of these contexts

Reply:
Good point. I've updated this.